templates/docs/examples/patterns/navigation/deprecated-dark.html

```diff
@@ -51,5 +51,6 @@
 
 <script>
   {% include 'docs/examples/patterns/navigation/_script.js' %}
+  initNavDropdowns('.p-subnav')
```

Comment:
When embedded it could use a new line and less indent (and a semicolon):

<img width="851" alt="Screenshot 2021-06-02 at 17 07 23" src="https://user-images.githubusercontent.com/83575/120505335-29ab5500-c3c5-11eb-9546-8a32c38039da.png">

```suggestion

initNavDropdowns('.p-subnav');
```

Same applies in other examples.